CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# avo-audit v1.0.0
+
```

Comment:
Should we have an `Unreleased` section at the top of the changelog that users can add to before it get's batched up in a release?

Reply:
Sounds good, added it on top